component/service/src/main/java/org/exoplatform/social/rest/impl/spacemembership/SpaceMembershipRestResourcesV1.java

```diff
@@ -320,10 +319,12 @@ public Response updateSpaceMembershipById(@Context UriInfo uriInfo,
         if (model.getStatus().equalsIgnoreCase(MembershipType.APPROVED.name())) {
           spaceService.addMember(space, targetUser);
           role = MembershipType.APPROVED.name();
+          LogUtils.logInfo("spaceMembership", "approve-space-invitation", "spaceName:" + spacePrettyName + ", receiver:" + targetUser, LogUtils.STATUS_OK, this.getClass());
```

Comment:
```suggestion
          LogUtils.logInfo("spaceMembership", "approve-space-invitation", "space_name:" + spacePrettyName + ",receiver:" + targetUser, LogUtils.STATUS_OK, this.getClass());
```
* Spaces are not avoided to avoid artifacts after the log parsing
* ``space_name`` should be used